README.md

```diff
@@ -94,78 +72,34 @@ export default {
 export const Playground = {};
 ```
 
-## Build
-
-### Vite
-
-Vite is used to build the local Storybook for testing and dev purposes
-
-### Rollup
-
-Rollup is used to build the addon for publishing
-
-```mermaid
-graph TD;
-    subgraph ADDON
-    A{{Rollup}}
-    B[index.ts]
-    C[manager.ts]
-
-    D[Panel addon]
-    E[Tool addon]
-
-    F[withPlayground]
-    U[generateAutocompletions]
-    G[PlaygroundPreview]
-    H[react-live]
+## Usage
 
-    I[Editor]
-    J[useCopyToClipboard]
-    K[usePlaygroundArgs]
-    Q[useInitialCode]
-    R[useBroadcastEditorChanges]
-    S[usePlaygroundState]
-    T[useEditorTheme]
-    L[useToolbarActions]
-    M[prettier]
-    N[react-codemirror]
+To use the Playground, navigate to the Storybook UI and select a story that has the playground decorator. In your addons panel, see an interactive code editor alongside your component, where you can modify the code and immediately see your changes reflected.
 
-    O[Toolbar icon]
+## Contributing
 
-    P[Storybook Addon API]
+Contributions are welcome! Feel free to open an issue or submit a pull request.
 
-    A -->|Entry| B
-    A -->|Entry| C
+### Develop locally
 
-    B -->|Exports| F
-    B -->|Exports| U
+Install dependencies and start
 
-    P --> D
-    P --> E
+```bash
+yarn
+yarn start
+# or
+npm i
+npm start
+```
 
-    E -->|Renders| O
+Go to [localhost:6006](http://localhost:6006)
 
-    C -->|Registers Addons| P
+### Build
 
-    F -->|Renders in a story| G
-    G ======>|Using lib| H
+#### Rollup
 
-    D -->|Calls| Q
-    D -->|Calls| R
-    R -->|Uses| S
-    D --->|Renders| I
-    I ====>|Using lib| N
-    I -->|Uses| L
-    I -->|Uses| K
-    I -->|Uses| T
-    I -->|Uses| J
+Rollup is used to build the addon for publishing.
```

Comment:
These also can be in a Architecture doc or something

Reply:
maybe in a CONTRIBUTION doc?

Reply:
Sounds good!